enterprise/backend/src/metabase_enterprise/sso/integrations/jwt.clj

```diff
@@ -62,11 +62,10 @@
   [user jwt-data]
   (when (sso-settings/jwt-group-sync)
     (when-let [groups-attribute (jwt-attribute-groups)]
```

Comment:
you mean `groups-attribute` ? it was not before so I changed it to use it instead of calling `(jwt-attribute-groups)` again.

Reply:
yeah, you could move the `(jwt-attribute-groups)` call to be inline since that's the only place we need to use the results. But that's mostly just a stylistic thing, up to you